src/lib/hooks/useBuyNowPrice.ts

```diff
@@ -0,0 +1,55 @@
+//- React Imports
+import { useQuery } from 'react-query';
+
+//- Hooks Imports
+import { useZnsSdk } from './useZnsSdk';
+
+export interface UseBuyNowPriceReturn {
+	buyNowPrice: string;
+	error: any;
+	isError: boolean;
+	isLoading: boolean;
+	isSuccess: boolean;
```

Comment:
Looks like the return type of `useQuery` hook. I am not sure these are useful or not

Reply:
Followed this up re conversation in discord. Much cleaner